README.md

```diff
@@ -24,6 +24,16 @@ If you need to install package deps from a source other than https://atom.io, su
 }
 ```
 
+If you need to install specific version of a package, you can suffix the package semver-compatible range with a `:`:
```

Comment:
"...you can add a semver-compatible range to the package name, separated by a `:`"